### PR TITLE
Multiple configure pipelines docs (plus tweaks)

### PR DESCRIPTION
**Pull request by richcooper95**
fixes [#42](https://github.com/syntasso/kratix-docs/issues/42)

### CODE REVIEW

File: docs/main/05-reference/04-promises/03-updates.md [hunk 1]
```diff
@@ -4,26 +4,56 @@ sidebar_label: Updates
...
+An update to a Promise will cause Kratix to reconcile on the new Promise definition,
+and any changes will be rolled out during this reconciliation.
+
+This may include:
+- Updating the Promise API, which rolls out an update to the underlying CRD for the
+  Resources managed by the Promise.
+- Updating the Promise or Resource workflows.
+- Updating the Promise scheduling.
+- Updating the Promise's static dependencies (the `dependencies` field in the Promise
+  spec).
```
Comment (kirederik) [inline docs/main/05-reference/04-promises/03-updates.md:18]: nothing against this addition, but is it adding any extra clarity (i.e. is "a promise update" something confusing that require examples?)

File: docs/main/05-reference/06-workflows.md [hunk 2]
```diff
@@ -83,67 +85,110 @@ Refer to the [Kubernetes Pod Spec
...
+                      secretKeyRef:
+                        name: # Secret name
+                        key: # Secret key
+```
 
 :::note
 
-Not all fields from the Pod spec are supported. We will add support for more
-fields in the future.
+The Secret must be accessible within the Pipeline's namespace.
```
Comment (kirederik) [inline docs/main/05-reference/06-workflows.md:123]: any reason to remove this?

File: docs/main/05-reference/04-promises/03-updates.md (hunk 1 above)
Comment (richcooper95) [reply docs/main/05-reference/04-promises/03-updates.md:18]: We've just come back to this and think it's probably worth keeping, if only to spell out to Kratix users what they can update

File: docs/main/05-reference/06-workflows.md (hunk 2 above)
Comment (richcooper95) [reply docs/main/05-reference/06-workflows.md:123]: Added it back with hindsight! It's useful to keep underneath the reference to the k8s Pod documentation 👍

**Pull request merged**